assets.mjs

```diff
@@ -0,0 +1,42 @@
+import sharp from "npm:sharp@0.33.5";
```

Comment:
Here if you merge this I think that will add me as a co-author. You can then make the file executable with `$ chmod +x assets.mjs`:

```suggestion
#!/usr/bin/env -S deno run -A
// @ts-check

import sharp from "npm:sharp@0.33.5";
```